configs/src/deployments/arbitrum.json

```
@@ -0,0 +1,184 @@
+{
+  "v1.3.0": {
+    "AddresslistVotingSetup": {
+      "address": "0x7a62da7B56fB3bfCdF70E900787010Bc4c9Ca42e",
+      "blockNumber": 145462271,
+      "deploymentTX": "0x6b427b56b4963a41da6a3e84b62b449480427823fb8f67e65357ae34122dc8b1"
```

Let's stick to camel case. 
```suggestion
      "deploymentTx": "0x6b427b56b4963a41da6a3e84b62b449480427823fb8f67e65357ae34122dc8b1"
```

---

done